internal/driver/glfw/window_test.go

```diff
@@ -864,7 +864,7 @@ func TestWindow_Clipboard(t *testing.T) {
 	assert.Equal(t, text, cb.Content())
 
 	// Restore clipboardContent, if any
-	cb.SetContent(cliboardContent)
+	cb.SetContent(cliboardContent) // this can fail but does not break the test, just means the CB is not restored.
```

Comment:
Is this comment still correct? It seems like it might have been addressed by the rest of the code...